src/main/java/JanggiApplication.java

```diff
@@ -0,0 +1,17 @@
+import config.MySqlConnector;
+import controller.JanggiController;
+import dao.JanggiDaoImpl;
+import dao.PieceDaoImpl;
+import service.JanngiService;
+import view.InputView;
+import view.OutputView;
+
+public class JanggiApplication {
+    public static void main(String[] args) {
+        final MySqlConnector mySqlConnector = new MySqlConnector();
+        final JanngiService janngiService = new JanngiService(new PieceDaoImpl(mySqlConnector),
+                new JanggiDaoImpl(mySqlConnector));
```

Comment:
사소하지만 이 부분은 한 줄로 두는 것이 조금 더 깔끔할 것 같아요. 😃 
```suggestion
        final JanngiService janngiService = new JanngiService(new PieceDaoImpl(mySqlConnector), new JanggiDaoImpl(mySqlConnector));
```